src/assets/v1.0.0/gnosis_safe.json

```diff
@@ -7,8 +7,7 @@
     "1": "0xb6029EA3B2c51D09a50B53CA8012FeEB05bDa35A",
     "4": "0xb6029EA3B2c51D09a50B53CA8012FeEB05bDa35A",
     "5": "0xb6029EA3B2c51D09a50B53CA8012FeEB05bDa35A",
-    "42": "0xb6029EA3B2c51D09a50B53CA8012FeEB05bDa35A",
-    "100": "0xb6029EA3B2c51D09a50B53CA8012FeEB05bDa35A",
```

Comment:
Why is Gnosis chain removed?

Reply:
It's seems was due a merge conflict. It's solved.